package.json

```diff
@@ -11,11 +11,15 @@
   },
   "private": true,
   "dependencies": {
-    "@grpc/grpc-js": "1.6.8",
-    "@protobuf-ts/grpc-transport": "2.7.0",
-    "axios": "0.27.2",
-    "object-hash": "3.0.0",
-    "tslib": "2.4.0"
+    "@buf/bufbuild_connect-web_open-feature_flagd-dev": "^1.7.2",
+    "@bufbuild/connect-web": "^0.2.1",
+    "@grpc/grpc-js": "^1.6.7",
+    "@protobuf-ts/grpc-transport": "^2.7.0",
+    "axios": "^0.27.2",
+    "node-cache": "^5.1.2",
+    "object-hash": "^3.0.0",
+    "ts-md5": "^1.3.1",
+    "tslib": "^2.3.0"
```

Comment:
@james-milligan I removed `ts-cache` and `cache` here - they seemed to be unused.

Reply:
i played around wioth both of these before `node-cache` and must have forgotten to remove them